package.json

```diff
@@ -20,7 +20,7 @@
     "@devvit/public-api": "0.11.17-next-2025-06-12-824bd74c5.0",
     "@devvit/server": "0.11.17-next-2025-06-12-824bd74c5.0",
     "@devvit/redis": "0.11.17-next-2025-06-12-824bd74c5.0",
-    "devvit": "0.11.17-next-2025-06-12-824bd74c5.0",
+    "devvit": "latest",
```

Comment:
I don't think we can do that for these because we rely on the latest CLI fixes for these to work. We can't in to next either since it may have regressions. The bolt template is the only one this can work on for now